src/reuse/modules/ui5/userInteraction.ts

```diff
@@ -249,6 +249,8 @@ export class UserInteraction {
         elem = await nonUi5.element.getByCss("[id='" + id + "'] input", index, timeout);
       }
       await elem.setValue(value);
+    }else{
```

Comment:
can you install prettier for vscode to indent the code correctly

Reply:
Ohh! My bad. Had disabled it in my vscode. The code is now formatted correctly